### PR TITLE
W 16518089 deploy json ch2plus duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/_partials/acb-deployment.adoc [hunk 1]
```diff
@@ -1,36 +1,139 @@
...
+| `1`
 
 | `autoStart`
-|  Indicates whether to start the application automatically after the deployment process completes. The default value is `true`.
-| CloudHub
+|  Indicates whether to start the application automatically after the deployment process completes. 
+| `true`
+
+| `loggingNgEnabled`
+| TODO_TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-deployment.adoc:32]: @dhairyashah-sf Can you help with the description here?

File: modules/ROOT/pages/_partials/acb-deployment.adoc [hunk 2]
```diff
@@ -1,36 +1,139 @@
...
+| `true`
+
+| `monitoringAutoRestart`
+| TODO_TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-deployment.adoc:36]: @dhairyashah-sf Can you help with the description here?

File: modules/ROOT/pages/_partials/acb-deployment.adoc [hunk 3]
```diff
@@ -1,36 +1,139 @@
...
+| `replicaSize`
+| Sets the size of each replica, specified in vCores. For more information, see xref:cloudhub-2::ch2-architecture.adoc#cloudhub-2-replicas[CloudHub 2.0 Replicas].
+| `0.1`
+|===
+// end::deploy-json-config-ch2[]
+//
+//
+// tag:deploy-json-config-ch2-example[]
+{
+  TODO_TODO_TODO
```
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-deployment.adoc:133]:
```suggestion
{
  "applicationName": "test-ch2-ds-deployment-runtime-test",
  "replicas": 1,
  "replicaSize": "0.1",
  "deploymentModel": "rolling", enum - rolling / recreate
  "desiredState": "STARTED", enum - STARTED / STOPPED
  "objectStoreV2": false // optional default - false
  "properties": {} // optional
}
```

@dhairyashah-sf When available, can you provide a better example of a CH2 deploy.json with default values I can use here?

File: modules/ROOT/pages/_partials/acb-deployment.adoc (hunk 2 above)
Comment (dukesphere) [inline modules/ROOT/pages/_partials/acb-deployment.adoc:36]:
```suggestion
| Restarts the deployed app if it is not responding.
```

Comment (dhairyashah-sf) [reply modules/ROOT/pages/_partials/acb-deployment.adoc:36]: @dukesphere Thanks for your question! 
I addressed this in our slack thread, sharing here for reference
_monitoringAutoRestart  - when enabled will restart the application if it is not responding [ref](https://docs.mulesoft.com/cloudhub/deploying-to-cloudhub#deploy-an-application-from-the-command-line)_

File: modules/ROOT/pages/_partials/acb-deployment.adoc (hunk 3 above)
Comment (dhairyashah-sf) [reply modules/ROOT/pages/_partials/acb-deployment.adoc:133]: @dukesphere sharing a sample **_deploy_ch2.json_** configuration file for CH2 deployments
`
{
  "applicationName": "test-application-deployment",
  "runtime": "4.7.3",
  "replicas": 1,
  "replicaSize": "0.1",
  "deploymentModel": "rolling",
  "desiredState": "STARTED",
  "objectStoreV2": false,
  "properties": {
    "sampleKey": "sampleValue"
  }
}
`
Do let me know if this looks good
Thanks!

File: modules/ROOT/pages/_partials/acb-deployment.adoc [hunk 4]
```diff
@@ -1,36 +1,139 @@
...
+  "properties": {
+    "sampleKey": "sampleValue"
+  },
+  "region": "us-east-2" 
+}
+// end:deploy-json-config-ch-example[]
+//
+//
+// tag::deploy-json-config-ch2[]
+For deployments to CloudHub 2.0 from Anypoint Code Builder, `deploy.json` supports the following properties.
```
Comment (dhairyashah-sf) [inline modules/ROOT/pages/_partials/acb-deployment.adoc:88]: We create and use a different file **_deploy_ch2.json_** for deployments to CloudHub 2.0

**Comment by dukesphere** [comment]
superceded by https://github.com/mulesoft/docs-code-builder/pull/385/files
Just FYI: @failup

**Pull request closed without merging**